core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateRangeIT.java

```diff
@@ -286,17 +287,22 @@ public void testSingleValueFieldWithStringDatesWithCustomFormat() throws Excepti
     }
 
     public void testSingleValueFieldWithDateMath() throws Exception {
+        int timeZoneOffset = randomIntBetween(-12, 12);
+        DateTimeZone timezone = DateTimeZone.forOffsetHours(timeZoneOffset);
+        String timeZoneSuffix = (timeZoneOffset == 0) ? "Z" : DateTime.now(timezone).toString("ZZ");
```

Comment:
Could you leave an example of what this will produce so anyone scanning doesn't have to hunt down the "ZZ" format?